site.yml

```diff
@@ -159,13 +159,37 @@
       tags: ['openstack', 'nova', 'data']
       when: not ironic.enabled|bool
 
+- name: ceph monitors
+  gather_facts: force
+  hosts: ceph_monitors
+  roles:
+    - role: ceph-common
+      tags: ['ceph', 'ceph-common']
+      when: ceph.enabled|bool
+
+    - role: ceph-monitor
+      tags: ['ceph', 'ceph-monitor']
+      when: ceph.enabled|bool
+
+- name: ceph osds
+  gather_facts: force
+  hosts: ceph_osds
+  roles:
+    - role: ceph-common
+      tags: ['ceph', 'ceph-common']
+      when: ceph.enabled|bool
+
+    - role: ceph-osd
+      tags: ['ceph', 'ceph-osd']
+      when: ceph.enabled|bool
+
 - name: cinder control plane
   gather_facts: force
   hosts: controller
   roles:
     - role: cinder-control
       tags: ['openstack', 'cinder', 'control']
-      when: cinder.enabled|bool
+      when: ceph.enabled|bool
```

Comment:
This should remain `cinder.enabled` since the default is coupled the `ceph.enabled`